puppet/modules/chassis/manifests/wp.pp

```diff
@@ -11,6 +11,8 @@
 	$admin_email    = 'admin@example.com',
 	$admin_password = 'password',
 	$network = false,
+	$plugins = [],
+	$theme = '',
```

Comment:
@shadyvb I think we should make it handle multiple wp.org themes. I know internally we'll rarely ever need to do that as we'll have submodules but I think other folks using Chassis might end up needing multiple themes from wp.org. Sorry to make it more difficult for you but we always try and think about all the use cases for Chassis 👍